Mathlib/Order/BoundedOrder.lean

```diff
@@ -89,6 +90,10 @@ theorem lt_top_of_lt (h : a < b) : a < ⊤ :=
 
 alias LT.lt.lt_top := lt_top_of_lt
 
+attribute [aesop (rule_sets := [finiteness]) unsafe 20%] ne_top_of_lt
+-- would have been better to implement this as a "safe" "forward" rule, why doesn't this work?
+-- attribute [aesop (rule_sets [finiteness]) safe forward] ne_top_of_lt
```

Comment:
@pitmonticone Can you consult with @JLimperg about whether there is a way to get this working?

Reply:
Sure, I'll get back to this tomorrow evening.

Reply:
```suggestion
attribute [aesop (rule_sets := [finiteness]) safe forward] ne_top_of_lt
```

Admittedly, the error message is extremely unhelpful.

Reply:
@JLimperg should I commit this now?

Reply:
I agree reverting this change is good, but the comment is now somewhat misleading - let's amend it to say something along the lines of your comment